interface/orders/orders_results.php

```diff
@@ -354,23 +354,23 @@ function validate(f) {
         $form_proc_type_desc = $ptrow['name'];
     }
 ?>
-    <?php xl('Procedure', 'e'); ?>:
+    <?php xlt('Procedure', 'e'); ?>:
    <input type='text' size='30' name='form_proc_type_desc'
     value='<?php echo addslashes($form_proc_type_desc) ?>'
     onclick='sel_proc_type()' onfocus='this.blur()'
-    title='<?php xl('Click to select the desired procedure', 'e'); ?>'
+    title='<?php xlt('Click to select the desired procedure', 'e'); ?>'
     style='cursor:pointer;cursor:hand' readonly />
    <input type='hidden' name='form_proc_type' value='<?php echo $form_proc_type ?>' />
 
-   &nbsp;<?php xl('From', 'e'); ?>:
+   &nbsp;<?php xlt('From', 'e'); ?>:
    <input type='text' size='10' class='datepicker' name='form_from_date' id='form_from_date'
     value='<?php echo $form_from_date ?>'
     title='<?php xl('yyyy-mm-dd', 'e'); ?>' />
 
-   &nbsp;<?php xl('To', 'e'); ?>:
+   &nbsp;<?php xlt('To', 'e'); ?>:
    <input type='text' size='10' class='datepicker' name='form_to_date' id='form_to_date'
     value='<?php echo $form_to_date ?>'
-    title='<?php xl('yyyy-mm-dd', 'e'); ?>' />
+    title='<?php xlt('yyyy-mm-dd', 'e'); ?>' />
```

Comment:
This is within an html attribute, so use xla. Also need to remove the `e` parameter and add the `echo`